cmd/channel/distributed/receiver/main.go

```diff
@@ -160,11 +162,15 @@ func flush(logger *zap.Logger) {
 // CloudEvent Message Handler - Converts To KafkaMessage And Produces To Channel's Kafka Topic
 func handleMessage(ctx context.Context, channelReference eventingchannel.ChannelReference, message binding.Message, transformers []binding.Transformer, _ nethttp.Header) error {
 
+	// Add a traceId to the logger so that we can more easily follow a particular message/failure in the log
+	traceLogger := logger.With(zap.String(logkey.TraceID, uuid.New().String()))
+	ctx = logging.WithLogger(ctx, traceLogger.Sugar())
+
```

Comment:
Minor nit - I wouldn't decorate the name `logger` as `tracelogger` just because we added the traceID to it.  Inconsistent with other instances below as well.

Reply:
I can name it something different, though I don't want to overwrite the package-global "logger" with the new logger containing this particular traceid